src/test/kotlin/uk/gov/justice/digital/hmpps/pollpush/services/CaseNoteListenerPusherIntTest.kt

```diff
@@ -16,7 +16,7 @@ import org.springframework.web.client.HttpServerErrorException
 import uk.gov.justice.digital.hmpps.pollpush.services.DeliusExtension.Companion.communityApi
 import uk.gov.justice.digital.hmpps.pollpush.services.health.IntegrationTest
 
-@ActiveProfiles("no-queues")
+@ActiveProfiles("z-no-queues")
```

Comment:
got caught out by https://github.com/spring-projects/spring-framework/issues/26004 so had to ensure that this profile gets applied last